include/zephyr/sys/util_internal.h

```diff
@@ -407,4 +407,785 @@
+#define UTIL_DEC_142 141
+#define UTIL_DEC_143 142
+#define UTIL_DEC_144 143
+#define UTIL_DEC_145 144
+#define UTIL_DEC_146 145
+#define UTIL_DEC_147 146
+#define UTIL_DEC_148 147
+#define UTIL_DEC_149 148
+#define UTIL_DEC_150 149
+#define UTIL_DEC_151 150
+#define UTIL_DEC_152 151
+#define UTIL_DEC_153 152
+#define UTIL_DEC_154 153
+#define UTIL_DEC_155 154
+#define UTIL_DEC_156 155
+#define UTIL_DEC_157 156
+#define UTIL_DEC_158 157
+#define UTIL_DEC_159 158
+#define UTIL_DEC_160 159
+#define UTIL_DEC_161 160
+#define UTIL_DEC_162 161
+#define UTIL_DEC_163 162
+#define UTIL_DEC_164 163
+#define UTIL_DEC_165 164
+#define UTIL_DEC_166 165
+#define UTIL_DEC_167 166
+#define UTIL_DEC_168 167
+#define UTIL_DEC_169 168
+#define UTIL_DEC_170 169
+#define UTIL_DEC_171 170
+#define UTIL_DEC_172 171
+#define UTIL_DEC_173 172
+#define UTIL_DEC_174 173
+#define UTIL_DEC_175 174
+#define UTIL_DEC_176 175
+#define UTIL_DEC_177 176
+#define UTIL_DEC_178 177
+#define UTIL_DEC_179 178
+#define UTIL_DEC_180 179
+#define UTIL_DEC_181 180
+#define UTIL_DEC_182 181
+#define UTIL_DEC_183 182
+#define UTIL_DEC_184 183
+#define UTIL_DEC_185 184
+#define UTIL_DEC_186 185
+#define UTIL_DEC_187 186
+#define UTIL_DEC_188 187
+#define UTIL_DEC_189 188
+#define UTIL_DEC_190 189
+#define UTIL_DEC_191 190
+#define UTIL_DEC_192 191
+#define UTIL_DEC_193 192
+#define UTIL_DEC_194 193
+#define UTIL_DEC_195 194
+#define UTIL_DEC_196 195
+#define UTIL_DEC_197 196
+#define UTIL_DEC_198 197
+#define UTIL_DEC_199 198
+#define UTIL_DEC_200 199
+#define UTIL_DEC_201 200
+#define UTIL_DEC_202 201
+#define UTIL_DEC_203 202
+#define UTIL_DEC_204 203
+#define UTIL_DEC_205 204
+#define UTIL_DEC_206 205
+#define UTIL_DEC_207 206
+#define UTIL_DEC_208 207
+#define UTIL_DEC_209 208
+#define UTIL_DEC_210 209
+#define UTIL_DEC_211 210
+#define UTIL_DEC_212 211
+#define UTIL_DEC_213 212
+#define UTIL_DEC_214 213
+#define UTIL_DEC_215 214
+#define UTIL_DEC_216 215
+#define UTIL_DEC_217 216
+#define UTIL_DEC_218 217
+#define UTIL_DEC_219 218
+#define UTIL_DEC_220 219
+#define UTIL_DEC_221 220
+#define UTIL_DEC_222 221
+#define UTIL_DEC_223 222
+#define UTIL_DEC_224 223
+#define UTIL_DEC_225 224
+#define UTIL_DEC_226 225
+#define UTIL_DEC_227 226
+#define UTIL_DEC_228 227
+#define UTIL_DEC_229 228
+#define UTIL_DEC_230 229
+#define UTIL_DEC_231 230
+#define UTIL_DEC_232 231
+#define UTIL_DEC_233 232
+#define UTIL_DEC_234 233
+#define UTIL_DEC_235 234
+#define UTIL_DEC_236 235
+#define UTIL_DEC_237 236
+#define UTIL_DEC_238 237
+#define UTIL_DEC_239 238
+#define UTIL_DEC_240 239
+#define UTIL_DEC_241 240
+#define UTIL_DEC_242 241
+#define UTIL_DEC_243 242
+#define UTIL_DEC_244 243
+#define UTIL_DEC_245 244
+#define UTIL_DEC_246 245
+#define UTIL_DEC_247 246
+#define UTIL_DEC_248 247
+#define UTIL_DEC_249 248
+#define UTIL_DEC_250 249
+#define UTIL_DEC_251 250
+#define UTIL_DEC_252 251
+#define UTIL_DEC_253 252
+#define UTIL_DEC_254 253
+#define UTIL_DEC_255 254
+
+#define UTIL_x2_0 0
```

Comment:
Nonblocking, but using a lowercase "x" here feels wrong to me. Shouldn't API macros be capitalized?